deploy/docker-ephemeral/docker-compose.yaml

```diff
@@ -13,13 +13,26 @@ services:
   fake_localstack:
     image: localstack/localstack:0.8.0  # NB: this is younger than 0.8.6!
     ports:
-      - 4569:4579 # ses
+      - 4569:4579 # ses # needed for local integration tests
       - 4575:4575 # sns
     environment:
       - DEBUG=1
       - DEFAULT_REGION=eu-west-1
       - SERVICES=ses,sns
 
+  basic_smtp: # needed for demo setup
+    # https://github.com/catatnight/docker-postfix
```

Comment:
is this image binding the listener to `0.0.0.0`?  if so, how do we avoid receiving spam from the outside world, and possibly relaying it?

Reply:
Good catch - people should use firewalls and not expose their dev setup to the world anyway, but I suppose we can provide better defaults for people who're not careful with their setup.

> is this image binding the listener to 0.0.0.0?

Yes, like all other docker containers started with docker-compose, it makes them accessible to the external world by default:

```
sudo netstat -antlp | grep LISTEN | grep 25
tcp6       0      0 :::25                   :::*                    LISTEN      3655/docker-proxy
```

It looks like this can be fixed by prefixing the exposed ports with `127.0.0.1:`, so after [this commit](84b08851c5749a020f1079d98dc8daf0227188bc):

```
sudo netstat -antlp | grep LISTEN | grep 25
tcp        0      0 127.0.0.1:25            0.0.0.0:*               LISTEN      28084/docker-proxy
```